package/kernel/linux/modules/netsupport.mk

```
@@ -1495,6 +1495,23 @@ endef
 $(eval $(call KernelPackage,inet-diag))
 
 
+define KernelPackage/inet-mptcp-diag
+  SUBMENU:=$(NETWORK_SUPPORT_MENU)
+  TITLE:=INET diag support for MultiPath TCP
+  DEPENDS:= +KERNEL_MPTCP +kmod-inet-diag
```

Please do not add the `KERNEL_MPTCP` configuration here, bgut depend on it:
```suggestion
  DEPENDS:= @KERNEL_MPTCP +kmod-inet-diag
```